project_parent/models/project.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2019 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class Project(models.Model):
+    _inherit = 'project.project'
+    _parent_store = True
+
+    parent_id = fields.Many2one(
+       comodel_name='project.project', string='Parent Project'
+    )
+    child_ids = fields.One2many(comodel_name='project.project',
+                                inverse_name='parent_id')
```

Comment:
These conflict with [fields in `project_wbs`](https://github.com/OCA/project/blob/1247db6a383ceb310403d787e9df00bb56dfe6af/project_wbs/models/project_project.py#L164), which point to `account.analytic.account`. So one of the modules has to change, I think. Changing this one would be less disruptive.

Reply:
changing it to parent_project_id